src/main/java/nerdhub/textilelib/event/client/render/hud/DrawHudCallback.java

```diff
@@ -10,14 +10,28 @@
 @Environment(EnvType.CLIENT)
 public interface DrawHudCallback {
 
-    Event<DrawHudCallback> EVENT = EventFactory.createArrayBacked(DrawHudCallback.class, listeners -> (type, hud, mouseX, mouseY, deltaTime) -> {
+    Event<DrawHudCallback> EVENT_BEFORE = EventFactory.createArrayBacked(DrawHudCallback.class, listeners -> (type, hud, deltaTime) -> {
```

Comment:
I'd rather see this implemented as subclasses, i.e. `DrawHudCallback$Pre` and `DrawHudCallback$Post`

Reply:
also the `Post` events should **not** be cancelable